### PR TITLE
Dev landing page

### PR DESCRIPTION
**Pull request by mahidhar1**
image
Added styles to Footer, Made the navbar dynamic with navbar links changing based on page. Also modified the menu icon button(responsive) action on popup open. Working on recruiter dashboard now on a separate branched named recruiter_dashboard.

### CODE REVIEW

File: src/common/Footer.jsx [hunk 1]
```diff
@@ -1,7 +1,119 @@
...
+      textAlign: 'center',
+      backgroundColor: '#34495E', 
+      color: '#FFFFFF', 
+    },
+    title: {
+        fontweight: 400, 
+    }, 
+    listOfItems: {
+        display: 'flex',
+        flexDirection: 'column',
```
Comment (n4alphapesto) [inline src/common/Footer.jsx:20]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/n4alphapesto/hirebook-frontend/pullRequest?prid=7756469)
```suggestion
        flexDirection: "column", 
```

File: src/containers/JobseekerDashboard.jsx [hunk 2]
```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import {Hidden, Link, Box, Grid, makeStyles, Typography } from '@material-ui/core';
+
+import FilterListIcon from '@material-ui/icons/FilterList';
+
+import { Navbar, Footer, PopUpComponent, StatsComponent, FilterComponent, SummaryComponent } from '../common';
+import oracle from '../assets/svg/oracle.svg';
```
Comment (n4alphapesto) [inline src/containers/JobseekerDashboard.jsx:7]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/n4alphapesto/hirebook-frontend/pullRequest?prid=7756469)
```suggestion
import oracle from "../assets/svg/oracle.svg";
```

File: src/common/Navbar.jsx [hunk 3]
```diff
@@ -1,81 +1,142 @@
...
-        
+        paddingLeft: theme.spacing(10),
     },
     navlinks: {
-        position: 'absolute',
-        right: '10%',
-        color: 'black',
-        
+        paddingRight: theme.spacing(10),
+        '& > *': {
```
Comment (n4alphapesto) [inline src/common/Navbar.jsx:23]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/n4alphapesto/hirebook-frontend/pullRequest?prid=7756469)
```suggestion
        "& > *": {
```

File: src/App.js [hunk 4]
```diff
@@ -1,19 +1,28 @@
 import './App.css';
 import { Provider } from 'react-redux';
-import store from './redux/store';
+import { ThemeProvider, createTheme } from '@material-ui/core/styles';
```
Comment (n4alphapesto) [inline src/App.js:3]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/n4alphapesto/hirebook-frontend/pullRequest?prid=7756469)
```suggestion
import { ThemeProvider, createTheme } from "@material-ui/core/styles";
```

File: src/common/Navbar.jsx [hunk 5]
```diff
@@ -1,81 +1,142 @@
...
+            margin: '20px',
+            padding: '20px 10px',
```
Comment (n4alphapesto) [inline src/common/Navbar.jsx:25]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/n4alphapesto/hirebook-frontend/pullRequest?prid=7756469)
```suggestion
        padding: "20px 10px",
```

File: src/components/Hero/Hero.jsx [hunk 6]
```diff
@@ -0,0 +1,46 @@
+import React from 'react';
```
Comment (n4alphapesto) [inline src/components/Hero/Hero.jsx:1]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/n4alphapesto/hirebook-frontend/pullRequest?prid=7756469)
```suggestion
import React from "react";
```

File: src/components/index.js [hunk 7]
```diff
@@ -0,0 +1,6 @@
+
+import TrustedCompanies from './TrustedCompanies/TrustedCompanies';
```
Comment (n4alphapesto) [inline src/components/index.js:2]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/n4alphapesto/hirebook-frontend/pullRequest?prid=7756469)
```suggestion
import TrustedCompanies from "./TrustedCompanies/TrustedCompanies";
```

File: src/components/TrustedCompanies/TrustedCompanies.jsx [hunk 8]
```diff
@@ -0,0 +1,141 @@
...
+        display: 'flex',
+        flexDirection: 'row',
+        flexWrap: 'wrap',
+        overflow: 'hidden',
+        backgroundColor: theme.palette.background.paper,
+    },
+    imageList: {
+        flexWrap: 'nowrap',
+        // Promote the list into his own layer on Chrome. This cost memory but helps keeping high FPS.
+        transform: 'translateZ(0)',
```
Comment (n4alphapesto) [inline src/components/TrustedCompanies/TrustedCompanies.jsx:43]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/n4alphapesto/hirebook-frontend/pullRequest?prid=7756469)
```suggestion
        transform: "translateZ(0)",
```

File: src/components/Testimonial/Testimonial.jsx [hunk 9]
```diff
@@ -0,0 +1,71 @@
+import React from 'react';
```
Comment (n4alphapesto) [inline src/components/Testimonial/Testimonial.jsx:1]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/n4alphapesto/hirebook-frontend/pullRequest?prid=7756469)
```suggestion
import React from "react";
```

File: src/common/index.js [hunk 10]
```diff
@@ -1,11 +1,25 @@
+import {Button} from './Button';
+import DashboardSkeleton from './DashboardSkeleton';
+import FilterComponent from './FilterComponent';
 import Footer from './Footer';
+import Image from './Image';
```
Comment (n4alphapesto) [inline src/common/index.js:5]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/n4alphapesto/hirebook-frontend/pullRequest?prid=7756469)
```suggestion
import Image from "./Image";
```

File: src/containers/RecruiterDashboard.jsx [hunk 11]
```diff
@@ -0,0 +1,94 @@
...
+
+
+
+const useStyles = makeStyles((theme) => ({
+    root: {
+        flexGrow: 1, 
+    },
+    button: {
+        background: "#79d4fd",
+        '&:hover': {
```
Comment (n4alphapesto) [inline src/containers/RecruiterDashboard.jsx:51]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/n4alphapesto/hirebook-frontend/pullRequest?prid=7756469)
```suggestion
        "&:hover": {
```

File: src/containers/index.js [hunk 12]
```diff
@@ -1,7 +1,13 @@
 import LandingPage from './LandingPage';
 import LoginPage from './LoginPage';
+import SignupPage from './SignupPage';
+import JobseekerDashboard from './JobseekerDashboard';
```
Comment (n4alphapesto) [inline src/containers/index.js:4]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/n4alphapesto/hirebook-frontend/pullRequest?prid=7756469)
```suggestion
import JobseekerDashboard from "./JobseekerDashboard";
```

File: src/common/Footer.jsx [hunk 13]
```diff
@@ -1,7 +1,119 @@
...
+                }
+
+            </div>
+            
+          </Grid>
+          <Grid item xs={12} sm={6} md={3}>
+            <Typography variant="h5" className={classes.title}>Jobs By Function</Typography>
+            <div className={classes.listOfItems}>
+                {
+                    ['Frontend', 'Backend', 'FullStack', 'Cloud'].map((role, i) => (
```
Comment (n4alphapesto) [inline src/common/Footer.jsx:63]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/n4alphapesto/hirebook-frontend/pullRequest?prid=7756469)
```suggestion
                    ["Frontend", 'Backend', 'FullStack', 'Cloud'].map((role, i) => (
```

File: src/components/TrustedCompanies/TrustedCompanies.jsx [hunk 14]
```diff
@@ -0,0 +1,141 @@
...
+        overflow: 'hidden',
+        scrollBehavior: 'smooth',
```
Comment (n4alphapesto) [inline src/components/TrustedCompanies/TrustedCompanies.jsx:45]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/n4alphapesto/hirebook-frontend/pullRequest?prid=7756469)
```suggestion
        scrollBehavior: "smooth",
```

File: src/components/TrustedCompanies/TrustedCompanies.jsx [hunk 15]
```diff
@@ -0,0 +1,141 @@
...
+import { useTheme } from '@material-ui/core/styles';
+import ArrowForwardIosIcon from '@material-ui/icons/ArrowForwardIos';
+import ArrowBackIosIcon from '@material-ui/icons/ArrowBackIos';
+
+import Image from '../../common/Image';
+
+
+
+import twilio from '../../assets/svg/twilio.svg';
+import disney from '../../assets/svg/disney.svg';
```
Comment (n4alphapesto) [inline src/components/TrustedCompanies/TrustedCompanies.jsx:12]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/n4alphapesto/hirebook-frontend/pullRequest?prid=7756469)
```suggestion
import disney from "../../assets/svg/disney.svg";
```

File: src/common/index.js (hunk 10 above)
Comment (n4alphapesto) [inline src/common/index.js:5]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/n4alphapesto/hirebook-frontend/pullRequest?prid=7756469)
```suggestion
import Image from "./Image";
```

File: src/components/Mission/Mission.jsx [hunk 16]
```diff
@@ -0,0 +1,107 @@
+import React from "react";
+
+import styled from "styled-components";
+import { Typography, Box, makeStyles, Grid } from "@material-ui/core";
+
+import valuesIcon from "../../assets/images/value.png";
+import visionIcon from "../../assets/images/vision.png";
+import missionIcon from "../../assets/images/mission.png";
+
+import './styles.css';
```
Comment (n4alphapesto) [inline src/components/Mission/Mission.jsx:10]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/n4alphapesto/hirebook-frontend/pullRequest?prid=7756469)
```suggestion
import "./styles.css";
```

File: src/common/index.js [hunk 17]
```diff
@@ -1,11 +1,25 @@
...
+import ImageGrid from './ImageGrid';
 import Navbar from './Navbar';
-import {Button} from './Button';
+import PopUpComponent from './PopUpComponent';
+import StatsComponent from './StatsComponent';
```
Comment (n4alphapesto) [inline src/common/index.js:9]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/n4alphapesto/hirebook-frontend/pullRequest?prid=7756469)
```suggestion
import StatsComponent from "./StatsComponent";
```

File: src/containers/index.js [hunk 18]
```diff
@@ -1,7 +1,13 @@
 import LandingPage from './LandingPage';
 import LoginPage from './LoginPage';
+import SignupPage from './SignupPage';
```
Comment (n4alphapesto) [inline src/containers/index.js:3]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/n4alphapesto/hirebook-frontend/pullRequest?prid=7756469)
```suggestion
import SignupPage from "./SignupPage";
```

File: src/containers/RecruiterDashboard.jsx [hunk 19]
```diff
@@ -0,0 +1,94 @@
...
+    }
+}
+
+    
+
+const jobPostData = {
+    type: 'companyDetails', 
+    companyLogo: oracle,
+    companyName: 'oracle', 
+    jobTitle: 'Frontend Developer',
```
Comment (n4alphapesto) [inline src/containers/RecruiterDashboard.jsx:36]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/n4alphapesto/hirebook-frontend/pullRequest?prid=7756469)
```suggestion
    jobTitle: "Frontend Developer", 
```

File: src/common/Navbar.jsx [hunk 20]
```diff
@@ -1,81 +1,142 @@
...
-                }
-            </Tabs>
+        <AppBar className={classes.root}>
+            <Toolbar className={classes.content}>
+                <Link href="/" className={classes.logo}>
+                    <Image src={logo} width={164}/>
+                </Link>
+            
+                <Hidden mdDown>
+                    <Box className={classes.navlinks} aria-label={`navbar `}>
```
Comment (n4alphapesto) [inline src/common/Navbar.jsx:67]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/n4alphapesto/hirebook-frontend/pullRequest?prid=7756469)
```suggestion
                    <Box className={classes.navlinks} aria-label={"navbar "}>
```

File: src/App.js (hunk 4 above)
Comment (n4alphapesto) [inline src/App.js:3]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/n4alphapesto/hirebook-frontend/pullRequest?prid=7756469)
```suggestion
import { ThemeProvider, createTheme } from "@material-ui/core/styles";
```

File: src/components/Testimonial/Testimonial.jsx [hunk 21]
```diff
@@ -0,0 +1,71 @@
...
+export default Testimonial;
+
+const Img = styled.img`
+    height: auto;
+    width: 100%;
+    max-height: 300px;
+`
+
+
+const useStyles = makeStyles(theme => ({
```
Comment (n4alphapesto) [inline src/components/Testimonial/Testimonial.jsx:65]: Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/n4alphapesto/hirebook-frontend/pullRequest?prid=7756469)
```suggestion
const useStyles = makeStyles((theme) => ({
```

File: src/components/Mission/Mission.jsx [hunk 22]
```diff
@@ -0,0 +1,107 @@
...
+        fontSize: 13,
+        paddingTop: 5,
+        textAlign: "center",
+
+    },
+    span_text: {
+        fontSize: 12
+    },
+    large: {
+        width: '100%',
```
Comment (n4alphapesto) [inline src/components/Mission/Mission.jsx:97]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/n4alphapesto/hirebook-frontend/pullRequest?prid=7756469)
```suggestion
        width: "100%",
```

File: src/components/index.js (hunk 7 above)
Comment (n4alphapesto) [inline src/components/index.js:2]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/n4alphapesto/hirebook-frontend/pullRequest?prid=7756469)
```suggestion
import TrustedCompanies from "./TrustedCompanies/TrustedCompanies";
```

File: src/components/TrustedCompanies/TrustedCompanies.jsx [hunk 23]
```diff
@@ -0,0 +1,141 @@
...
+    useEffect(() => {
+        
+        const interval = setInterval(() => {
+            //console.log(scrollRef.current.scrollLeft, scrolled, direction)
+            if(direction === "r"){
+                setScrolled(scrolled => scrolled + logoWidth)
+                scrollRef.current.scrollLeft += logoWidth;
+            } 
+            if(direction === "l") {
+                setScrolled(scrolled => scrolled - logoWidth)
```
Comment (n4alphapesto) [inline src/components/TrustedCompanies/TrustedCompanies.jsx:86]: Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/n4alphapesto/hirebook-frontend/pullRequest?prid=7756469)
```suggestion
                setScrolled((scrolled) => scrolled - logoWidth)
```

File: src/components/Hero/Hero.jsx [hunk 24]
```diff
@@ -0,0 +1,46 @@
...
+    height: auto;
+    min-height: 500px;
+    background-size: cover;
+
+    @media (max-width: 768px) {
+        background-image: url('${HeroImageSm}');
+    }
+`;
+
+const useStyles = makeStyles(theme => ({
```
Comment (n4alphapesto) [inline src/components/Hero/Hero.jsx:38]: Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/n4alphapesto/hirebook-frontend/pullRequest?prid=7756469)
```suggestion
const useStyles = makeStyles((theme) => ({
```

File: src/components/TrustedCompanies/TrustedCompanies.jsx [hunk 25]
```diff
@@ -0,0 +1,141 @@
...
+
+    const executeScrollLeft = () => {
+        //console.log(scrollRef.current.scrollLeft, direction)
+        setScrolled(scrolled => scrolled - logoWidth)
+        scrollRef.current.scrollLeft -= logoWidth;  
+    }
+
+    const executeScrollRight = () => {
+        //console.log(myRef.current.scrollLeft, direction)
+        setScrolled(scrolled => scrolled + logoWidth)
```
Comment (n4alphapesto) [inline src/components/TrustedCompanies/TrustedCompanies.jsx:108]: Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/n4alphapesto/hirebook-frontend/pullRequest?prid=7756469)
```suggestion
                setScrolled((scrolled) => scrolled + logoWidth)
```

File: src/components/TrustedCompanies/TrustedCompanies.jsx [hunk 26]
```diff
@@ -0,0 +1,141 @@
...
+                setDirection("r")
+            }     
+        }, 1000);
+        return () => clearInterval(interval);
+        
+    }, [scrolled, direction, logoWidth])
+
+    const executeScrollLeft = () => {
+        //console.log(scrollRef.current.scrollLeft, direction)
+        setScrolled(scrolled => scrolled - logoWidth)
```
Comment (n4alphapesto) [inline src/components/TrustedCompanies/TrustedCompanies.jsx:102]: Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/n4alphapesto/hirebook-frontend/pullRequest?prid=7756469)
```suggestion
                setScrolled((scrolled) => scrolled - logoWidth)
```

**Review by n4alphapesto — APPROVED** [review]

**Comment by n4alphapesto** [comment]
Initial dev changes

**Review by rajtejani — APPROVED** [review]

**Pull request merged**